### DISCUSSION TITLE
Story of a mage

### DISCUSSION THREAD

**Discussion by Coryis** [chromiecraft/chromiecraft#4188] [Roleplay]
_The room was completely dark, only a torch in the corridor in front of it cast a glimmer on the cold, black stone blocks and let the metal of the cages glitter. Not that it would have made any difference to the neglected inmates of these cages, who sat mute and completely emaciated on the floor._

_The only emotion that still showed on the sunken face was fear. Fear of the echo of boots approaching this godforsaken room deep beneath the ruins of Lordaeron._

_In the glow of a torch, a hunched figure entered the room, his skin distorted in death into a grim face. He looked around the room, yellow eyes gazing ominously. The only person present, a man, remained sitting cross-legged in his apathy at the opening of his one-feet large cage._

"You seem to deserve my attention, unlike that worthless filth otherwise piled in this hole." "Here," _he offered him a piece of stale bread_ "you don't want your "light" to hear your prayers until you're already rotting on your excrement, do you? - Well, I see you haven't completely lost your mind yet. So, listen carefully to what I'm about to tell you."

_The figure smoothed her immaculate russet robes as he sat down on a chair that stood against the wall._

"As you can easily imagine, I once belonged to your people. I was born the son of simple farmers in Tarren's mill before the orc invasion. My parents were poor, and their knowledge amounted to arithmetic to ten and signing with three Xen. I was an only child, my brothers all died before I was born, their graves were behind the small house. As expected of a good farmer's son and the only "heir" to the farm, I helped as best I could with the work in the fields. However, inexplicable phenomena kept happening around me, something hovered in the air, a bale of hay caught fire, the meager oatmeal for breakfast froze. Soon the rumor spread among the naive farmers that I was cursed or possessed. People began to avoid me; the other children did not want to play with me in the rare free time and the adults gave my parents a wide berth. So, it came to pass that on one of the many occasions I pointed angrily at an older boy, a ball of fire struck his face and burned the flesh. I ran away just as frightened as the hit; however, it should be clear what thereupon in the place for an uproar prevailed. But before, as is not uncommon in the area, an angry mob tied me to a stake at the stake, a traveling mage of Dalaran became aware of the stories about a bewitched boy and investigated them.

I had just passed my eighth birthday, when in the afternoon the same mage in his magnificent robes suddenly stood in front of me in the field to get a picture of the bewitched boy. I remember clearly that he cast a spell on me, and completely incredulously repeated it repeatedly. Afterwards he smiled at me and asked if I was the cause of those scary stories in the village and had cursed the blacksmith's son. In my childish fear, which I had of the magicians, who turned innocent people into animals on a whim, I answered in the affirmative. He laughed and told me for the first time in my life that I was special and that I had to accompany him to Dalaran to become a mage. Imagine that: An eight-year-old farmer's son should go to the magnificent Dalaran that no one in the village ever saw and when the rumors made the rounds was only fearfully spoken of in whispers, and I should learn there! I felt as if someone had given me a blow. After all, everyone was afraid of me, and now I knew two things: this was nothing bad and they were right in their fear.

Of course, I wanted to travel with him right away, but in my childish naivety I had forgotten that my parents had to give their consent. To my great surprise, they even did, but not as you likely believe. It had nothing to do with the great chance for the poor child or charity. Because of the mood in the village my parents had hardly any money left because they thought the curse would be transferred to those who ate fruits from our field. And now you had an opportunity to get rid of me and their bad reputation. Yes, your look tells me you are on the right track. That is right, they sold me for the price of two horses. Fortunately, or described from his point of view, unfortunately, the compassion had seized him, and he paid the proud sum. I got to hear it from my sleeping place against the wall facing the shed and did not know how my parents could do such a thing. I packed my meager things still in the same night and saw my parents for the last time.

For my parents received the fate you deserved. My father killed my mother in a drunken stupor because of the wealth, when he got remorse a little later. He himself was murdered by a thief when he bragged about his wealth in the village pub. I learned it only years later and did not cry a tear after you.

The following years were hard. For from one outsider role, I fell into the next. If I was an outsider first because of my abilities, this now applied to my origin. Fortunately, my mentor took care of me and introduced me to the secrets of the arcane arts. I hardly left the libraries or classrooms for years.
Thanks to my mentor's guidance, I learned quickly, but found that the atmosphere in Dalaran was just like that among the peasants in Tarren's Mill, only on a different level. The atmosphere was one of resentment and jealousy, everyone wanted to get behind the research results and secrets of the other.

Since my mentor was a proven eccentric, I received the unrestrained harshness of ridicule, especially during his travels. A "peasant magician," as my peers called me, is not the rule, but the exception, especially in a family that did not have a single magically gifted member.

Despite these problems, I managed to become a reasonably accepted apprentice of the arcane arts. I learned of the first war against the orcs late, only the appearance of a magical anomaly far to the south made the rounds. In this my mentor died, and from now on my opinion of the ivory tower dwellers sank from day to day, to the same extent as my opinion of the rulers. Since I was on my own from now on, who would take an unpleasant pupil of an eccentric under his wing, I had more free space for myself and occupied myself with the political events outside Dalaran. I followed the second attack of the orcs with great interest, but despite my existing education and sufficient age, I was not allowed to join the troops of the Alliance and support them. I was beside myself with anger. What was the use of all this power, all this knowledge, if you are condemned to inactivity and merely watch? After my initial anger had faded, I turned even more to the political events outside the small cosmos of Dalaran and was shocked. These small-minded rulers were already fighting over the smoking remains of their lands, barely after the blood had dried on their swords. Why don't the Kirin Tor use their power to end the squabbles of these intellectual yokels and put you where you belong? My frustration grew and I voiced my opinion among my peers. This was a big mistake. To my further dismay, many of you were already so attuned to the scheming system that I was soon publicly admonished and on the verge of banishment. But just at this time of my doubts, a younger apprentice joined me, and I got into conversation with him.

We talked about politics and the misdeeds of the small-minded powerful. He agreed with me to a great extent, and after a long time and wine, he informed me that many among the boys, in view of the recent wars were speaking out for a change in the policies of Dalaran and the kingdoms and were organizing in secret. He offered to introduce me to someone who held similar views as I did and wanted to make me an offer. I agreed to a meeting, which took place on the northern coast of Tirisfal.

None of my superiors objected to getting rid of me, at least temporarily. The only regret was that a young, talented mage joined a rebel like me.

So, one night I met an inconspicuous person on the north coast of Tirisfal. He told me that the rulers and their light were using the population for their own purposes and were thinking only of themselves and their sinecures. Something had to be done against this. Since Dalaran was also loyal to you, the new era of the shadow would have to be established from the underground. A new empire should arise, led by the new, powerful Kirin Tor. Only through their wisdom and arcane strength could be prevented that in a renewed war against the orcs, dwarves or others, a catastrophe would be avoided.

Perhaps the name 'Cult of the Damned' should have made me wonder, but I was young, politically inexperienced and convinced by his idea, which was remarkably like mine. Especially since he promised me something that no one had ever achieved before: immortality. The mysterious leader did not reveal himself to me but told me to return to Dalaran and gather more support, he would call me if he needed me. So it was that I became one of the cogs in Kel'Thuzad's preparations for war against Lordaeron. The following time I was busy gathering the disaffected around me and introducing them to the cult. A group of several dozen gathered, who were mainly busy searching for the darkest secrets of their mentors. But I had not only a political function. I was given formulas to learn and teach to the others. I was trained in things that exceeded my wildest imagination and had this been known, would likely have meant my death. But finally, I had the power to change history myself and to establish a new era with the Cult of the Damned. Well, I was not that wrong.
Because of my rapid rise in the ranks of the organization, my thinking changed. It was not just any cultists who were to receive power after our victory, but a cultist: me. The faint whispers in the back of my mind encouraged me in my actions, one of the many disadvantages of being in the service of the damned Scourge.

Soon I received the call for a particularly honorable task. I was to guard one of the plague cauldrons together with other cultists below me. Everyone involved was clear about what these artifacts would do and welcomed the murder of the population to get material for our army to overthrow the king and the council. We guarded the cauldron as slowly the plague took more and more of the population and joined the army of the scourge.
Do not look so horrified. Yes, you are right, I have helped to almost wipe out my own people and to rise again as a perversion of life. Do you expect repentance from me now? Such a weak existence deserves nothing else, not even the loss of your homeland can bring about a change in your way of thinking. But now continue.

As a reward for my loyalty, I was one of the first to receive the gift of immortality, though not in the form I imagined. In joyful anticipation, when the time came, I entered the ritual circle and drank the liquid contained in the crystal vial. At first, I felt nothing, but soon the effect took place and every corner of my body felt as if I were on fire. Mad with pain, I collapsed and writhed on the floor. Death came quickly, but not quickly enough if I had my way. My last thought now of my death was the realization that I and all others of the Cult of the Damned had been betrayed. But I had no time to think further about it, because an overwhelming presence tore down all my psychic barriers of my still active mind and poured into the last corner. My self, or what remained, was locked into the furthest dark corner.

From then on, I was part of an all-encompassing collective, a continuous dream without redeeming awakening. Only single images remained to me as admonishing memories, the Sunwell, Stratholme, Dalaran. I slaughtered countless as part of the army. Old, children, humans, elves, it was meaningless. The only thing that mattered was the fulfillment of the Scourge's will. I fought alongside ghouls, death knights. But after the storming of Lordaeron, the dream cracked, the barriers that suppressed myself were able to leak out again and I regained control. However, the presence was still there, clinging to every available niche, now my mind had become a battlefield. My last memory was the magnificent sight of Sylvanas after the second conquest of Lordaeron, after which I fell into a comatose state, directing all reserves to my own final liberation.

After overcoming even, the last reverberation of alien control, I awoke in a tomb in Tirisfal. I had regained control of myself and of my body, which had been frozen in death. But a sudden shock hit me when I tried to remember a formula: I could not remember. There was nothing, a huge black hole in my memories. The price of freedom was high. With all my will I forced myself to concentrate and indeed I was able to laboriously channel a shred of arcane energies into one of the easiest spells I could remember, a fireball. I was overjoyed. Seeing the rat die inside the tomb meant that I had only lost my knowledge, not my ability to channel arcane energies. Alarmed by the squeaking of the rat, a guard hurried over, who explained to me the basics of the new situation in the world. Thus, I learned to which person I owed my existence and my freedom and swore to dedicate my existence to him from now on.

I regret that even now, after intensive study, I can command only a part of my power, but this is insignificant in view of the eternity I possess to restore or relearn even the last remnant of my knowledge. For I know that I will rule this world under the dark princess after all you mortals, whether Alliance or Horde, lie rotting on the battlefields or have joined us. Soon our plague will sweep you away like a desert storm sweeps away the grains of sand. But first, the Scourge must be destroyed to secure our freedom once and for all.

You must be wondering why I am telling you all this. Why I am revealing my life story to you and entrusting you with your darkest fears about the intentions of the Forsaken and our dark princess.

It could be because you will not live to see the next morning. Just as I once took a concentrated form of the Scourge's plague from a crystal vial, you have taken the latest creation of the Apothecarium in the loaf of bread. I wanted to make sure you knew exactly who was responsible for your death. Your death is called Coryis Blackmoore, the traitor to his own people!"

_The Forsaken Mage looked deeply into the human's eyes once again, revealing his hatred._

"You are inferior creatures, neither your narrow-mindedness nor that of the other races will ever stand between us and our freedom or existence. Perhaps we will meet again tomorrow...."

_The light of the torch had long since disappeared back into the hallway, as Blackmoore's laughter still echoed through the chamber._

**Comment by ghost** [comment]
Nice and long story. I wonder, though, if this story belongs in the category [**[Show and tell]**](https://github.com/chromiecraft/chromiecraft/discussions/categories/show-and-tell), but I don't know if that category is meant only for in-game examples or not.

**Reply by FrancescoBorzi** [reply]
we can create a specific category for RP

**Comment by Coryis** [comment]
Schall I delete it and post it where else?

**Reply by ghost** [reply]
Hang on until the Discussions Categories are updated. No need to move it until there is a new RP category in the list. 😄
- https://github.com/chromiecraft/chromiecraft/discussions

**Reply by FrancescoBorzi** [reply]
no need to remove it @Coryis !

**Comment by Yehonal** [comment]
I love this story! GJ. I'm moving this one to our new role-play board
